cmd/search/main.go

```diff
@@ -268,7 +312,7 @@ func (o *options) MetadataFor(path string) (Result, error) {
 		if err != nil {
 			return result, fmt.Errorf("expected path bugs/bug-NUMBER: %s", path)
 		}
-		result.Name = fmt.Sprintf("Bug %d", id)
+		result.Name = fmt.Sprintf("Issue %d", id)
```

Comment:
This seems suspect...it's inside a `bugs/` block of code?  Why switch it to `Issue`?

Reply:
Most likely some IDE refactoring went wrong. Fixed.